plonky3/src/circuit_builder.rs

```diff
@@ -46,6 +28,8 @@ pub(crate) struct PowdrCircuit<'a, T> {
     witness: Option<&'a [(String, Vec<T>)]>,
     /// Callback to augment the witness in the later stages
     _witgen_callback: Option<WitgenCallback<T>>,
+    /// Identity of all public cells
```

Comment:
Do we need this field? Can't it be computed from the other fields and therefore be a method?